setup.py

```diff
@@ -67,7 +67,7 @@ def get_version():
     "setuptools",
     "sseclient-py>=1.7.2,<2",
     "sse-starlette>=0.10.3,<1",
-    "starlette>=0.20.4,<0.27",
+    "starlette>=0.24.0,<0.27",
```

Comment:
@benjaminpkane is this okay?

Reply:
I guess its just a tightening, so no downstream consequences huh?

Reply:
Yes, we recently had the request to increase upper bound. We may have users dependent on the lower bound but it's hard to say. I'm ok with it